tests/snuba/api/endpoints/test_project_trace_item_details.py

```diff
@@ -82,15 +81,29 @@ def test_simple(self):
         trace_details_response = self.do_request("logs", item_id)
 
         assert trace_details_response.status_code == 200, trace_details_response.content
+
+        timestamp_nanos = int(self.one_min_ago.timestamp() * 1_000_000_000)
         assert trace_details_response.data["attributes"] == [
             {"name": "bool_attr", "type": "bool", "value": True},
             {"name": "log.severity_number", "type": "float", "value": 0.0},
             {"name": "tags[bool_attr,number]", "type": "float", "value": 1.0},
             {"name": "tags[float_attr,number]", "type": "float", "value": 3.0},
             {"name": "tags[int_attr,number]", "type": "float", "value": 2.0},
+            # this is stored as a float for searching, so it is not actually very precise
+            {
+                "name": "tags[sentry.timestamp_precise,number]",
+                "type": "float",
+                "value": pytest.approx(float(timestamp_nanos), abs=1e12),
+            },
```

Comment:
Do we want column definitions for these? https://github.com/getsentry/sentry/blob/bf1a5f13f619abd64a417a836604f6a90176e1c9/src/sentry/search/eap/spans/attributes.py#L209-L218